website/pages/en/index.js

```diff
@@ -105,9 +105,6 @@ class Index extends React.Component {
               the more confidence they can give you.
             </i>
           </p>
-          <MarkdownBlock>
```

Comment:
It would be cool to have this animate a transition between the different libraries

Reply:
Can do this as a followup